db/rdbms/src/main/java/io/camunda/db/rdbms/write/utils/NumberChecker.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.db.rdbms.write.utils;
+
+public class NumberChecker {
+
+  private static final String INTEGER_REGEX = "^-?\\d+$";
+  private static final String DOUBLE_REGEX = "^-?\\d*[.,]?\\d+$";
```

Comment:
This does not cover scientific notation of Doubles like "1.1e7". DO we need to cover it?

Reply:
@romansmirnov @danielkelemen ?

Reply:
A, I dont use this class. :D Was my first IMPL before using apache commons.